src/TRoom.h

```diff
@@ -270,25 +270,29 @@ inline QDebug operator<<(QDebug debug, const TRoom* room)
     }
 
     QMap<QString, QList<QPointF>> customLines = room->customLines;
+    QMap<QString, QColor> customLinesColor = room->customLinesColor;
+    QMap<QString, bool> customLinesArrow = room->customLinesArrow;
+    QMap<QString, Qt::PenStyle> customLinesStyle = room->customLinesStyle;
     if (!customLines.isEmpty()) {
-        debug.nospace() << ", customLines=(";
-        for (QMap<QString, QList<QPointF>>::const_iterator it = customLines.begin(); it != customLines.end(); ++it) {
-            debug.nospace() << it.key() << ": " << it.value() << ", ";
+        debug.nospace() << ", customlines=(";
+        for (auto it = customLines.constBegin(); it != customLines.constEnd(); ++it) {
+            debug.nospace() << it.key().toLower() << ": " << it.value() << " (color: " << customLinesColor.value(it.key()).name().toLower()
```

Comment:
Don't `toLower()` the `it.key()` it ***is*** case-sensitive (it can even be a Lua script!) and discarding that information may be significant in debuggery... 😉

Reply:
Even you example shows this:
>Output when doing qDebug() << room;
TRoom(4), name="", area=2, pos=0,0,0, exits:, specialExits=("**climb Rope**".5, ), customlines=("**climb rope**": QList(QPointF(4.5,5.5), QPointF(4.5,9.5), QPointF(6,9.5)) (color: "#808000", arrow: no, style: 5), ), weight=1

Reply:
Fixed.